File: new/atoms/smtp-hosts.test.mjs
import * as m from './smtp-hosts.mjs';
const SNAP = {"SMTP_HOSTS":"{\"gmail.com\":\"smtp.gmail.com:465\",\"googlemail.com\":\"smtp.gmail.com:465\",\"outlook.com\":\"smtp-mail.outlook.com:587\",\"hotmail.com\":\"smtp-mail.outlook.com:587\",\"yahoo.com\":\"smtp.mail.yahoo.com:465\",\"walla.co.il\":\"out.walla.co.il:465\"}"};
let f = 0;
for (const [k, s] of Object.entries(SNAP)) if (JSON.stringify(m[k]) !== s) { console.error('✗ ' + k + ' סטה מהצילום'); f = 1; }
if (f) process.exit(1); console.log('✓ smtp-hosts: צילום-ערך תואם — ירוק');

File: new/dart-maor/smtp-hosts_test.dart
// בדיקת-חוזה (רתמת-זהב) · smtp-hosts — צילום-ערך jsonEncode≡JSON.stringify.
// חוללה מכנית (gen-data-dart). הרצה: dart run --enable-asserts ⇒ OK
import 'dart:convert';
import 'smtp-hosts.dart';

void main() {
  // ‏1) צילום-ערך: jsonEncode ≡ JSON.stringify של המקור (זהה-ביט).
  if (jsonEncode(smtpHosts) != '{"gmail.com":"smtp.gmail.com:465","googlemail.com":"smtp.gmail.com:465","outlook.com":"smtp-mail.outlook.com:587","hotmail.com":"smtp-mail.outlook.com:587","yahoo.com":"smtp.mail.yahoo.com:465","walla.co.il":"out.walla.co.il:465"}') {
    throw StateError('FAIL smtpHosts: צילום-הערך סטה');
  }
  print('OK smtp-hosts: 1 data-snapshot(s) passed');
}
